homeassistant/components/deconz/.translations/en.json

```diff
@@ -17,16 +17,23 @@
                 "description": "Do you want to configure Home Assistant to connect to the deCONZ gateway provided by the Hass.io add-on {addon}?",
                 "title": "deCONZ Zigbee gateway via Hass.io add-on"
             },
-            "init": {
+            "user": {
                 "data": {
-                    "host": "Host",
-                    "port": "Port"
+                    "host": "Select discovered deCONZ gateway",
+                    "manual_input": "Configure deCONZ gateway manually"
                 },
-                "title": "Define deCONZ gateway"
+                "title": "Select deCONZ gateway"
             },
             "link": {
                 "description": "Unlock your deCONZ gateway to register with Home Assistant.\n\n1. Go to deCONZ Settings -> Gateway -> Advanced\n2. Press \"Authenticate app\" button",
                 "title": "Link with deCONZ"
+            },
+            "manual_input": {
```

Comment:
We should rename the two keys for host/port when merging this so that translators don't have to translate them again.

Reply:
Aren't host and port the most common keys?